### DISCUSSION TITLE
Form表单能否加个属性，允许submit提交的时候不要自动校验全部字段

### DISCUSSION THREAD

**Discussion by h25302** [arco-design/arco-design-vue#1805] [Q&A] [answered]
- [ ] I'm sure this does not appear in [the issue list of the repository](https://github.com/arco-design/arco-design-vue/issues)

## What problem does this feature solve?
如果表单字段设置了服务器异步校验规则，在表单submit提交的时候，是不需要所有字段自动校验的。
目前在表单提交的时候，每个字段都会向服务器发送异步校验，导致发生了很多无效的网络交互。

## What are the similar cases of this feature
希望Form表单能否增加一个属性开关，允许submit提交的时候不要自动校验全部字段。
允许提交后，我们可以手动控制提交后的处理，由1次网络交互即可处理全部字段的检验。

**Comment by hehehai** [comment, marked as answer]
@h25302 hi, thx。表单验证这里目前是不太灵活，这里官方团队也有在尝试做重构处理，但短期内不会发布新的改动，期待下一个大的版本。
